src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -2470,6 +2482,12 @@ private void OnBoardingMenuGuide_Click(object sender, RoutedEventArgs e)
             }
         }
 
+        private void FileTrustWarning_Click(object sender, RoutedEventArgs e)
+        {
+            var dynViewModel = DataContext as DynamoViewModel;
+            dynViewModel.FileTrustWViewModel.ShowWarningPopup = true;
```

Comment:
`dynViewModel.FileTrustWViewModel?.ShowWarningPopup` just in case

Reply:
@QilongTang I think the "?" cannot used in a left-handed expression when assigning the value, as in this case, then I had to check if is not null directly, see the changes in next commit:
https://github.com/DynamoDS/Dynamo/pull/12916/commits/6e3af78f9f27e36771c3ceb54e8ea240bf3c9047